examples/detection/detr_coco/box_ops.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+"""
+Utilities for bounding box manipulation and GIoU.
+"""
+import torch
+from torchvision.ops.boxes import box_area
+
+
+def box_cxcywh_to_xyxy(x):
```

Comment:
All these box conversion operations are ported to torchvision since 0.8.1.
you can use [`torchvision.ops.box_convert`](https://pytorch.org/docs/stable/torchvision/ops.html) function. We use this widely in our library too and support from torchvision 0.8.1.

We can avoid all these code and use
`boxes = box_convert(boxes, in_fmt="cxcywh", out_fmt="xyxy")`